### PR TITLE
Add Intel vs. Arm blog post

### PR DESCRIPTION
**Pull request by ntolia**
## Description

This uses Corso/kopia microbenchmarks to compare the performance of Intel vs. ARM
architectures

### CODE REVIEW

**Comment by ntolia** [comment]
I am assuming we want to push this out on Monday, right?

**Comment by serverless-mom** [comment]
Yes i think Monday would be best.

File: website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+of different chunks of identical data, even if they're not the same
+size or don't start at the same point in the dataset.
+
+The size of the blocks and the algorithm used to divide the dataset
+are important considerations in deduplication, as they can affect the
+speed and efficiency of deduplication. For these results we compare
+the performance of Buzhash vs. the Rabin-Karp algorithm with block
+sizes of 1MiB, 2MiB. 4MiB, and 8MiB.
+
+![Throughput of Corso splitters](./images/benchmarking/splitter_througput.png)
```
Comment (serverless-mom) [inline website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md:127]: Screen Shot 2022-12-09 at 8 29 27 AM
We either need to style these with a background or add some kind of bg to the .png, this isn't super readable

**Review by serverless-mom — COMMENTED** [review]
need to update the graphics for readability.

**Comment by ntolia** [comment]
> need to update the graphics for readability.

Added a background to the images. Take a look. Happy to play with opacity and color options but trying to balance both light and dark mode is hard for these graphs.

**Comment by serverless-mom** [comment]
New version looks much more readable, thanks!

File: website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+`SHA3` HMAC variants for hashing but ARM starts to outperform with
+parallelism of 4 and above. This once again highlights the
+applicability of ARM to multi-threaded applications.
+
+## Conclusion
+
+The above results show how exciting ARM is as an architecture for us
+data-processing nerds because of the performance and cost
+optimizations possible. The improvement is a result of the hardware
+acceleration available on the ARM platform we used for benchmarking
```
Comment (gmatev) [inline website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md:228]:
```suggestion
acceleration available on the ARM platform used for benchmarking
```

File: website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+(AWS Graviton) and the optimization found in the libraries used in our
```
Comment (gmatev) [inline website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md:229]:
```suggestion
(AWS Graviton) and the additional optimizations in the libraries used in our
```

File: website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+product. While these things will constantly shift, we're excited
```
Comment (gmatev) [inline website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md:230]:
```suggestion
software. While these things may constantly shift, we're excited
```

**Review by gmatev — COMMENTED** [review]
Just some minor suggestions in the last paragraph. Optional if you want to take them

**Comment by ntolia** [comment]
> Just some minor suggestions in the last paragraph. Optional if you want to take them

@gmatev I think I accepted all your suggestions but please do confirm.

File: website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+
+Within these results, we will focus on the numbers with parallelism
+set to 8 and will focus on throughput numbers (higher is better)
+measured in MiB/s. For easier visualization, we also present the
+relative performance where we compare the throughput of ARM
+vs. Intel. For a more price-accurate comparison, we also present the
+price-adjusted relative performance where we see how many more MiB/s
+we can process for the same dollar.
+
+While there is color on some of the results not presented but give us
```
Comment (ashmrtn) [inline website/blog/2022-12-12-benchmarking-intel-vs-arm-for-backup-applications.md:106]:
```suggestion
While the results shown here are truncated for brevity, feel free to give us
```

**Pull request merged**